InternxtDesktop/Views/Backup/FolderSelectorView.swift

```diff
@@ -55,7 +55,7 @@ struct FolderSelectorView: View {
                                     if (!urls.contains(url)) {
                                         try self.backupsService.addFoldernameToBackup(
                                           FoldernameToBackup(
-                                              url: url.absoluteString,
+                                              url: url.relativePath,
```

Comment:
why do we need to use a relative path? if the app location changes, this is not going to work as is relative right?